templates_jinja2/event_details.html

```diff
@@ -106,19 +106,21 @@ <h3 class="panel-title">Event in progress</h3>
         </div>
         <div class="clearfix"></div>
 
-        {% if matches_upcoming %}
-        <div class="col-sm-6">
-          <h4>Upcoming Matches</h4>
-          {{ mtm.upcoming_match_table(matches_upcoming) }}
-        </div>
-        {% endif %}
-        {% if matches_recent %}
-        <div class="col-sm-6">
-          <h4>Recent Matches</h4>
-          {{ mtm.recent_match_table(matches_recent) }}
+        <div id="liveevent-content">
```

Comment:
What're your thoughts on having a graceful degradation back to the old view if firebase doesn't have the necessary data?  Or is that not a concern?  🤔

Reply:
If there is no javascript, the old view won't get replaced. If there is, it will get replaced. Firebase will have the necessary data to render last/upcoming matches, so it will look like the old view but live updating.

Reply:
Fair enough.  👍